mlflow/R/mlflow/.build-doc.R

```diff
@@ -0,0 +1,14 @@
+# Install Rd2md from source as a temporary fix for the rendering of code examples, until
+# a release is published including the fixes in https://github.com/quantsch/Rd2md/issues/1
+# Note that this commit is equivalent to commit 6b48255 of Rd2md master
+# (https://github.com/quantsch/Rd2md/tree/6b4825579a2df8a22898316d93729384f92a756b)
+# with a single extra commit to fix rendering of \link tags between methods in R documentation.
+devtools::install_git("https://github.com/smurching/Rd2md", ref = "mlflow-patches")
```

Comment:
Shall we use commit hash instead of branch which may be changed ?

Reply:
Good point, but we usually don't update this branch.